URA-695/correlation_plots.py

```
@@ -0,0 +1,231 @@
+    # Create the Site column
+    vcf_df['site'] = vcf_df['CHROM'].astype(str) + ':' \
+                    + vcf_df['POS'].astype(str)  + '_' \
+                    + vcf_df['REF'].astype(str) + '/' \
+                    + vcf_df['ALT'].astype(str)
+
+    # Create the AF
+    vcf_df['AF'] = vcf_df['SAMPLE'].str.split(':', expand=True)[2]
+
+    # Create the DP
+    vcf_df['DP'] = vcf_df['INFO'].str.split(';').apply(
+                   lambda x: ''.join((y for y in x if y.startswith('DP='))))
```

Ruff (0.8.0)

Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)